openedx/core/djangoapps/cookie_metadata/middleware.py

```diff
@@ -0,0 +1,59 @@
+"""Middleware to changes name of an incoming cookie"""
+from django.conf import settings
+
+
+class CookieNameChange:
+    """Changes name of an incoming cookie"""
+
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        """
+        Changes the names of a cookie in request.COOKIES
+
+        For this middleware to run:
+        - set COOKIE_NAME_CHANGE_ACTIVATE_EXPAND_PHASE =  True
+        - COOKIE_NAME_CHANGE_EXPAND_INFO is a dict and has following info:
+            "old_name": Previous name of cookie
+            "new_name": New name of cookie
+            if you also want to delete the cookie in user's browser, set "old_domain" as well
```

Comment:
I don't see any code corresponding to this line.

Reply:
Yep, this was for code that was recently removed